test/emqtt_quic_SUITE.erl

```diff
@@ -50,6 +52,39 @@ t_quic_sock(Config) ->
     ok = emqtt_quic:close(Sock),
     quic_server:stop(Server).
 
+t_0_rtt(Config) ->
+    Port = 4568,
+    SslOpts = [ {cert, certfile(Config)}
+              , {key,  keyfile(Config)}
+              , {idle_timeout_ms, 10000}
+              , {server_resumption_level, 2} % QUIC_SERVER_RESUME_AND_ZERORTT
+              , {peer_bidi_stream_count, 10}
+              , {alpn, ["mqtt"]}
+              ],
+    Server = quic_server:start_link(Port, SslOpts),
+    timer:sleep(500),
+    {ok, {quic, Conn, _Stream}=Sock} = emqtt_quic:connect("localhost",
+                                                          Port,
+                                                          [ {alpn, ["mqtt"]}, {active, false}
+                                                          , {quic_event_mask, 1}
+                                                          ],
+                                                          3000),
+    send_and_recv_with(Sock),
+    ok = emqtt_quic:close(Sock),
+    NST = receive
+              {quic, nst_received, Conn, Ticket} ->
+                  Ticket
+          end,
+    {ok, Sock2} = emqtt_quic:connect("localhost",
+                                     Port,
+                                     [{alpn, ["mqtt"]}, {active, false},
+                                      {nst, NST}
```

Comment:
Is there a way to check that it really used 0-RTT instead of a fresh new connection?  Perhaps assert that another NST is not received?

Reply:
Sorry I forget to click `comment` button before resolving the conversation.

The only way to check if NST is in use is by checking Wireshark trace.

if 0-RTT failed, it will be fallback to 1-RTT without application notice.

The client will receive yet another NST after 0-RTT success and this is by the server impl. .